docs/src/pages/getting-started/supported-platforms/supported-platforms.md

```diff
@@ -5,12 +5,13 @@
 ## Browser
 
 Material-UI supports the latest, stable releases of all major browsers and platforms.
-It also supports Internet Explorer 11.
 You don't need to provide any JavaScript polyfill as it manages unsupported browser features internally and in isolation.
 
-| IE  | Edge  | Firefox | Chrome | Safari | Googlebot |
-| :-- | :---- | :------ | :----- | :----- | :-------- |
-| 11  | >= 14 | >= 52   | >= 49  | >= 10  | ✅        |
+| Edge  | Firefox | Chrome | Safari |
+| :---- | :------ | :----- | :----- |
+| >= 83 | >= 68   | >= 83  | >= 13  |
```

Comment:
Nice, it means we can drop:

- https://github.com/mui-org/material-ui/blob/3e0ce3e29f68fb1b6b433589e6fbf2d91a08927b/packages/material-ui/src/Button/Button.js#L331-L337
- https://github.com/mui-org/material-ui/blob/3e0ce3e29f68fb1b6b433589e6fbf2d91a08927b/packages/material-ui/src/Slider/Slider.js#L130-L135

Reply:
Not so sure about the flexbox bug but I'll look again through the codebase for older polyfills. They should be part of this PR.

Reply:
Or let's go over them when going over built-ins